attributes/default.rb

```diff
@@ -192,3 +192,37 @@
 ]
 default['repose']['uri_identity']['group'] = 'User_Standard'
 default['repose']['uri_identity']['quality'] = 0.5
+
+default['repose']['api_validator']['cluster_id'] = ['all']
+default['repose']['api_validator']['multi_role_match'] = true 
+default['repose']['api_validator']['delegating_quality'] = 0.6 
+default['repose']['api_validator']['validators'] = [
```

Comment:
Looks like the defaults are specific to cloud feeds and require a lincense. Can we make them a little more generic? Feeds should be able to set up their validators in whatever role or wrapper cookbook they're using.